src/System Application/App/Azure Blob Services API/src/ABSOperationResponse.Codeunit.al

```diff
@@ -93,6 +93,24 @@ codeunit 9050 "ABS Operation Response"
         exit(Values[1]);
     end;
 
+    [NonDebuggable]
+    procedure GetMetadataValueFromResponseHeaders(): Dictionary of [Text, Text]
+    var
+        Metadata: Dictionary of [Text, Text];
+        Headers: HttpHeaders;
+        HeaderKeys: List of [Text];
+        HeaderKey: Text;
+        Values: array[100] of Text;
```

Comment:
Is there a reason this needs to be 100?

Reply:
No there is No particular reason , as the array need to be provided with size and for safer side i have given the number 100